webserver/views/api/v1/core.py

```diff
@@ -109,3 +111,48 @@ def _validate_offset(offset):
     else:
         offset = 0
     return offset
+
+@bp_core.route("/low-level", methods=["POST"])
+def submit_low_level_nombid():
```

Comment:
In the last commit I implemented all your suggestion, the only missing thing is the redirect. I don't understand what you mean, the client in the current implementation expects to receive only the gid(mbid,msid) to complete the dataset dictionary, so why do we have to redirect?